app/manager-frontend/src/pages/ActivitySource/index.jsx

```diff
@@ -164,13 +186,15 @@ export const ActivitySource = ({ activitySource, action, id }) => {
                   }
                   hasEmptyDefault
                 />
-                {isUpdate ? null : <Alert status='info'>
-                  <AlertIcon />
-                  Result Modifiers can be added after an Activity Source is created.
-                </Alert>}
-                <HStack>
+                {isUpdate ? null : (
+                  <Alert status="info">
+                    <AlertIcon />
+                    Result Modifiers can be added after an Activity Source is
+                    created.
+                  </Alert>
+                )}
+                <HStack justify={"space-between"}>
                   <Button
-                    w="200px"
                     leftIcon={<FaArrowRight />}
```

Comment:
When this is a `Save` button, the icon should be a floppy disk; the arrow should only be for the "create" version.